packages/story-editor/src/components/publishModal/mainContent/index.js

```diff
@@ -21,56 +21,88 @@ import {
   Button,
   BUTTON_SIZES,
   BUTTON_TYPES,
-  Text,
 } from '@googleforcreators/design-system';
 import styled from 'styled-components';
 import { __ } from '@googleforcreators/i18n';
+import { useMemo } from '@googleforcreators/react';
 /**
  * Internal dependencies
  */
 import { MANDATORY_INPUT_VALUE_TYPES } from '../types';
+import useInspector from '../../inspector/useInspector';
+import { DOCUMENT_PANEL_NAMES } from '../../panels/document';
+import { HEADER_BAR_HEIGHT, HEADER_BAR_MARGIN } from '../constants';
 import MandatoryStoryInfo from './mandatoryStoryInfo';
 import StoryPreview from './storyPreview';
 
 const Main = styled.div`
   display: grid;
   min-width: 917px;
   width: 100%;
-  height: 100%;
-  margin: 0 -3.7% 0 3.7%;
-  grid-gap: 3.7%;
-  /* grid-template-columns: 230px 276px 308px; */
-  grid-template-columns: 25% 30% 34%;
+  height: calc(100% - ${HEADER_BAR_MARGIN} - ${HEADER_BAR_HEIGHT});
+  /* grid-template-columns: 281px 313px 326.5px; */
+  grid-template-columns: 30.5% 34% 35.5%;
   grid-template-rows: auto;
   grid-template-areas:
     'preview mandatory panel'
-    'footer footer panel';
+    'footer mandatory panel';
 `;
 
 const _StoryPreview = styled.div`
   grid-area: preview;
-  padding-top: 20px;
+  margin: 20px 18px 0 32px;
 `;
 
 const _MandatoryStoryInfo = styled.div`
   grid-area: mandatory;
+  padding: 0 18px;
+  overflow: scroll;
+
+  & > section {
+    border: none; /* Override the default border that is part of the base panel structure since this is destructured */
+  }
 `;
 
-const PanelContainer = styled.div`
+const PanelContainer = styled.div.attrs({ role: 'tabpanel' })`
   grid-area: panel;
+  height: 100%;
+  margin-left: 18px;
   background-color: ${({ theme }) => theme.colors.bg.secondary};
+  border-bottom-right-radius: ${({ theme }) => theme.borders.radius.medium};
+  overflow: scroll;
 `;
 
 const Footer = styled.div`
   grid-area: footer;
+  display: flex;
+  align-items: flex-end;
+  margin: 0 18px 20px 32px;
 `;
 
+const WP_STATUS_PANEL = 'status'; // Unsure what the best way to grab this value is, if any, from wp-story-editor
```

Comment:
This is my outstanding question for this. It feels wrong to have a hardcoded string that references the status (visibility) panel in WordPress when so much work has been done to separate packages. Should this come from config???